src/utils/httpUtils.ts

```diff
@@ -1,6 +1,8 @@
+import { ORG_ID_REGEXP } from '../constants';
+
 export function getURL(baseURL: string, path: string, orgId?: string): string {
   if (routeToGateway(baseURL)) {
-    if (!orgId) {
```

Comment:
q: I dimly recall there being some support for passing org slugs instead of IDs. Is this related/what are the implications?

Reply:
Hmm so what are the implications there? I assume the API gateway doesn't support org slugs anymore? I understand this is only FedRAMP right now, but does that mean that if we were to, in theory, enable it for non-FedRAMP environments we would no longer support org slugs?

Reply:
Exactly. api-gateway routing only supports org id's, and not the slugs. Once all the other environments will be using the api-gateway as a sole means of entry to the cluster, we will require this check everywhere, unless api-gateway changes the way it handles incoming requests (enables org slugs).

Reply:
Okay, but to be clear: We can't do that in existing environments. If my understanding is correct, we would have a bunch of customers relying on org slugs working and the API gateway migration would break all of them. We'd have to make sure we transform the org slug into an org id before we make the request.

Not a blocker for this PR specifically, but it may be worth document some of the discussions we had here in a comment :)

Reply:
In the existing environments the routing doesn't use orgId at all. We are normally sending requests to endpoints such as `/filters` or `/bundle`, directly to deeproxy. The api-gateway expects the request url to contain the orgId always. This is the reason the url is created differently for FedRamp. Once the api-gateway will be used for other environments, the url will have to be constructed in the same way.

Reply:
Also, as far as I know, org slugs are not used right now for request url generation, but I may be wrong. For sure they are not used for code analysis in the code-client library.

Reply:
I am aware, I was talking about once we migrate the existing environments :)